accounting/receivables/customer_payments/online_payment.rst

```diff
@@ -0,0 +1,42 @@
+======================
+Invoice Online Payment
+======================
+
+To make it more convenient for your customers to pay the invoices you issue, you can activate the
+**Invoice Online Payment** feature, which adds a *Pay Now* button on their **Customer Portal**.
```

Comment:
"Allow your customer to just not see, but also pay their invoices through their Customer Portal. Offering extra convenient payment options, you show flexibility and appeal to a wider audience."

Reply:
Thanks, I added a sentence based on your suggestion :)